### PR TITLE
Add tests for centos7 and amazonlinux, add back travis ci for ansible-galaxy content scoring

### PR DESCRIPTION
**Pull request by rmoriar1**
These tests were failing due to b/181554122. Adding them back now that has been resolved.

Also add back .travis.yml file since Ansible galaxy uses it generate its content score shown here: https://galaxy.ansible.com/googlecloudplatform/google_cloud_ops_agents

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
 - name: Validate Operating Systen
   assert:
-    that: ansible_os_family == 'Windows' or ansible_distribution in ['Debian', 'Ubuntu', 'RedHat', 'CentOS', 'SLES', 'openSUSE', 'SuSE', 'SLES_SAP', 'Windows']
+    that: >
+      ansible_os_family == 'Windows' or ansible_distribution in ['Debian', 'Ubuntu', 'RedHat', 'CentOS', 'Amazon', 'SLES', 'openSUSE', 'SuSE', 'SLES_SAP',
+      'Windows']
     msg: >
       Received invalid Operating System: '{{ ansible_distribution }}'. The Cloud Ops Ansible role supports the following OSs: 'Debian', 'Ubuntu', 'RedHat',
-      'CentOS', 'SLES', 'openSUSE', 'SuSE', 'SLES_SAP' and 'Windows'.
+      'CentOS', 'Amazon', 'SLES', 'openSUSE', 'SuSE', 'SLES_SAP' and 'Windows'.
```
Comment (qingling128) [inline tasks/main.yml:9]: The Logging and Monitoring Agents support Amazon EC2 VMs. But Ops Agent does not support Amazon EC2 VMs. I guess the tests might fail for Ops Agent.

Comment (rmoriar1) [reply tasks/main.yml:9]: Yup, I've added an exclude option to our test matrix so the ops-agent tests never run with AmazonLinux2.
https://github.com/GoogleCloudPlatform/google-cloud-ops-agents-ansible/pull/52/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR62

**Comment by rmoriar1** [comment]
@qingling128 ping :)

**Review by qingling128 — APPROVED** [review]
LGTM

File: tasks/main.yml (hunk 1 above)
Comment (qingling128) [reply tasks/main.yml:9]: Oops, missed the last email update. Just LGTM'd. Thanks for adding this!

**Pull request merged**